frontend/animal/Experiment.js

```diff
@@ -110,6 +102,15 @@ class Experiment {
             .addFooter("")
             .show({maxWidth: 1000});
     }
+
+    render($div) {
```

Comment:
@rabstejnek fixed regression here in #268; the `render` method is required for showing endpoint modals on the heatmaps